libsolidity/interface/StandardCompiler.cpp

```diff
@@ -451,8 +447,7 @@ Json::Value StandardCompiler::compileInternal(Json::Value const& _input)
 	{
 		errors.append(formatErrorWithException(
 			_exception,
-			false,
-			"UnimplementedFeatureError",
```

Comment:
I think we shouldn't remove the exception type since this is passed down in the standard JSON output.

Reply:
It's a bit of an implementation detail leaking, but I agree we shouldn't change the JSON interface without careful consideration.